tests/unit/test.linter.js

```diff
@@ -21,6 +22,13 @@ import {
   assertHasMatchingError,
 } from './helpers';
 
+// Increasing jest timeout when running on nodejs 10,
+// due to some intermittency when running this tests
+// on Travis.
+if (process.release.lts === 'Dubnium') {
```

Comment:
Not sure if it's any easier, but you can also do Node version sniffing using `process.version`:

```sh
node -e "console.log(process.version)" # v14.11.0

nvm use 10 # Now using node v10.21.0 (npm v6.14.4)
node -e "console.log(process.version)" # v10.21.0
```

Reply:
@pdehaan eh, I was wondering where they did put that :-)

Reply:
Even worse answer if you want to check for specific Node versions using **semver** module:

```js
const semver = require("semver");

const version = process.version.replace(/^v/, "");

if (semver.lt(version, "11.0.0")) {
  console.log(`${version} is too old, increase test timeout.`);
} else {
  console.log(`Congrats, ${ version} is neat!`);
}
```